R/errors.R

```diff
@@ -3,3 +3,21 @@ hintr_error <- function(message, error, status_code = 400L, ...) {
   pkgapi::pkgapi_stop(message, error, errors = NULL, status_code = status_code,
                       key = key, ...)
 }
+
+hintr2_404_handler <- function(req, res) {
```

Comment:
The difference between this and the default pkgapi 404 handler is that is adds a key to each of the errors and has a translated error message